terms.html

```diff
@@ -143,6 +143,13 @@
   </dd>
 
 
+  <dt><dfn data-lt="">DID resolver</dfn></dt>
```

Comment:
```suggestion
  <dt><dfn data-lt="">DID resolver|DID resolvers</dfn></dt>
```

Reply:
Sorry this doesn't make sense to me, what are you trying to do here?

Reply:
I think this was probably supposed to be inside the `data-lt` attribute

Reply:
@rhiaro - Thank you, yes.
@peacekeeper - Suggestion revised.

Reply:
I see, but I believe this is not necessary.. ReSpec should take care of plural forms automatically.

Reply:
ReSpec docs do suggest that it can handle (most) plurals (but apparently not optionals [i.e., `(s)`]), which means there are several others to be deleted from the terms.html doc, the presence of which others were what suggested to me the need for this one.

I don't have a horse in this race, aside from consistency within the doc.

Reply:
Fixed @TallTed's concern in https://github.com/w3c/did-spec/pull/82/commits/c1dbcfe4f41680dea3fc468d81eb783d5ffd665c